components/schemes/Container/src/MDCContainerScheme.h

```diff
@@ -33,6 +33,7 @@
  */
 @property(nonatomic, nullable, readonly) id<MDCTypographyScheming> typographyScheme;
 
+@optional
```

Comment:
Please do not make `shapeScheme` optional. This would go against our decisions for Container Scheme where all types are required (if nullable). It also doesn't have any effect on NSCopying support, since if the property were non-nil, it should still be copied if appropriate.